locust/test/test_main.py

```diff
@@ -2348,7 +2348,7 @@ def t(self):
             output.extend(stdout.splitlines())
             output.extend(stdout_worker.splitlines())
 
-            self.assertIn("Shutting down (exit code 0)", "\n".join(output))
+            self.assertIn('Shutting down (exit code 0)', "\n".join(output))
```

Comment:
If you haven't already done so, I recommend installing the ruff plugin in vscode, that way you'll get formatting whenever you save the file. And set up the pre-commit hook as well, so any linting errors will be caught at commit-time: https://pre-commit.com/